integration/jwt.go

```diff
@@ -0,0 +1,33 @@
+package integration
+
+import (
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/infobloxopen/atlas-app-toolkit/auth"
+)
+
+const (
+	// TestSecret dummy secret used for signing test JWTs
+	TestSecret = "some-secret-123"
```

Comment:
Do we need this public?

Reply:
Nope. Also, I need to comb through this code with `go vet` and `golint` and fix any issues that come up.